tests/sentry/backup/test_correctness.py

```diff
@@ -126,10 +55,19 @@ def import_then_export(tmp_path: Path, fixture_file_name: str) -> None:
 
     res = validate(input, output)
     if res.findings:
-        raise AssertionError(res.pretty())
+        raise ValidationError(res)
 
 
-@pytest.mark.django_db(transaction=True, reset_sequences=True, databases="__all__")
+@django_db_complete_reset
 @freeze_time("2023-06-22T23:00:00.123Z")
-def test_fresh_install(tmp_path):
-    import_then_export(tmp_path, "fresh-install.json")
+def test_good_fresh_install_validation(tmp_path):
+    import_export_then_validate(tmp_path, "fresh-install.json")
+
+
+@django_db_complete_reset
+def test_bad_fresh_install_validation(tmp_path):
+    try:
+        import_export_then_validate(tmp_path, "fresh-install.json")
+    except ValidationError as e:
```

Comment:
if we're expecting a `ValidationError` to be raised, can we use a context manager here to ensure things are cleaned up correctly? something like `assertRaises`?

Reply:
Done, using `pytest.raises`.